spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -167,9 +167,12 @@ def get_parser():
         '-lrootlet',
         metavar=Metavar.file,
         help=textwrap.dedent("""
-            Dorsal nerve rootlets segmentation. Example: `anat_rootlets.nii.gz`
-            Only labels within the range C2-C8 are supported. If labels outside this range are provided, they will be ignored.
-            Each value corresponds to the spinal level (e.g.: 2 for spinal level 2). If you are using more than 2 labels, all spinal levels covering the region of interest should be provided (e.g., if you are interested in levels C2 to C7, then you should provide spinal level labels 2,3,4,5,6,7).
+            Dorsal (and ventral) nerve rootlets segmentation. Example: `anat_rootlets.nii.gz`
+
+            Both dorsal-only as well as dorsal and ventral rootlets are supported.
```

Comment:
I have not tested this specific scenario where only dorsal rootlets are provided, since now in the PAM50 we have dorsal and ventral, maybe we should test or omit this, what do you think @valosekj ?

Reply:
I removed "dorsal-only" in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4934/commits/a2cd18121cf4f0851ad08cc926af0f63f5c6ce59. I think we can merge